talks/_posts/2020-6-16-flannery-cpdd-presentation.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: talk
+title: "Peer influences on adolescents’ intentions for future electronic nicotine delivery system use"
+nickname: 2020-6-16-flannery-cpdd-presentation
+authors: "Flannery JS"
+year: "2020"
+conference: "College on Problems of Drug Dependence"
+image: /assets/images/talks/Flannery_cpdd_2020.png
+projects: []
+tags: []
+
+# Content
+slides:
+video: https://www.youtube.com/watch?v=-cOLufeebek
+
+# Links
+doi: https://www.youtube.com/watch?v=-cOLufeebek
```

Comment:
you can drop this one

Reply:
the doi?

Reply:
yeah, since it's not a permanent digital object identifier, and because you already have the link to the video in the video field above.